styles.css

```diff
@@ -251,3 +280,193 @@ textarea {
   background-color: #2230d2;
   color: #fff;
 }
+
+@media (min-width: 600px) {
```

Comment:
- [ ] Please make sure the range of the desktop view is between `992px` and `1100px`, as per requirement.